### DISCUSSION TITLE
PI 12 Iteration 1

### DISCUSSION THREAD

**Discussion by jzbmw** [eclipse-tractusx/item-relationship-service#382] [General]
GOAL: Prepare for Umbrella HELM & Data Sov. Guardrails

## Minister Security
@dsmf 
Timebox: 2h
- [ ] Check Veracode for findings
- [ ] Check Invict for findings
- [ ] run and chack Dashtool IP

## Release Champion: 
@ds-psosnowski 
Timebox: 2h

- [ ] Has the Release been conducted (Release is being skipped!)
- [ ] Check for Dependabots PR's on Tractus-X

## Docu-Guru:
@ds-ext-kmassalski  
Timebox: 2h 
- [ ] Focused on Documentation of Tickets

## Testmanager
@ds-alexander-bulgakov 
- [ ] Update [Release 24.03 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [ ] Update Test Status - PI11 [Test Status - PI11](https://confluence.catena-x.net/display/PL/Test+Status+-+PI11)
- [ ] Update [Integration Test - PI12 - Release 2024.05](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI12+-+Release+2024.05)

## Firefighter
@ds-jhartmann 
- [ ] Maintenance of our community
- [ ] Response to Github issues https://github.com/eclipse-tractusx/item-relationship-service/issues
- [ ]  To be defined, eg: Onboarding of new colleague, quality-gates, escalation topics
- [ ] Communicating and discussing with external contributors in Eclipse Matrix Chat

**Comment by ds-alexander-bulgakov** [comment]
- [x] Update [Release 24.05 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.05+----+Content+Overview) 
- [x] Update [Test Status - PI12](https://confluence.catena-x.net/pages/viewpage.action?pageId=114072227) Test Status - PI12
- [x] Update [Integration Test - PI12 - Release 24.05](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI12+-+Release+2024.05)
